Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For TOTEM, the alignment is quite delicate, thus we try to run simulations in conditions as close to the real data taking as possible.

The reason for having several copies of RP_Detectors_Assembly_* is also related to simulation. At the time when we designed the code, Geant4 geometry was build from "CompactView" representation of the geometry. Therefore, if there was only one RP_Detectors_Assembly file, the internal structure of each RP would be forced to be the same. And we want to have simulations with a more precise model.